web/src/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "lib": ["es6", "dom"],
+    "moduleResolution": "node",
```

Comment:
I don't believe `moduleResolution` is needed per https://www.typescriptlang.org/v2/en/tsconfig#moduleResolution

I removed and ran `yarn fmt` and tsc did not yell at me.